rdfunit-junit/src/main/java/org/aksw/rdfunit/junit/RLOGStatement.java

```diff
@@ -0,0 +1,42 @@
+package org.aksw.rdfunit.junit;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import com.hp.hpl.jena.rdf.model.ResourceFactory;
+import org.aksw.rdfunit.tests.results.RLOGTestCaseResult;
+import org.aksw.rdfunit.tests.results.TestCaseResult;
+import org.junit.runners.model.Statement;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class RLOGStatement extends Statement {
+
+    private final RdfUnitJunitStatusTestExecutor rdfUnitJunitStatusTestExecutor;
+    private final RdfUnitJunitTestCase child;
+
+    RLOGStatement(RdfUnitJunitStatusTestExecutor rdfUnitJunitStatusTestExecutor, RdfUnitJunitTestCase child) {
+        this.rdfUnitJunitStatusTestExecutor = rdfUnitJunitStatusTestExecutor;
+        this.child = child;
+    }
+
+    @Override
+    public void evaluate() throws Throwable {
+        final Collection<TestCaseResult> testCaseResults = rdfUnitJunitStatusTestExecutor.runTest(child);
+        final Collection<RLOGTestCaseResult> remainingResults = new ArrayList<>();
+        for (TestCaseResult t : testCaseResults) {
+            RLOGTestCaseResult r = (RLOGTestCaseResult) t;
+            if (!child.getInputModel().contains(ResourceFactory.createResource(r.getResource()), null)) {
```

Comment:
We could make this a little more explicit using a local variable. Just to understand why this is done, e.g.

```java
boolean resourceIsPartOfInputModel = child.getInputModel().contains(ResourceFactory.createResource(r.getResource()), null);
if(!resourceIsPartOfInputModel) ...
```